api/desecapi/models.py

```diff
@@ -227,3 +230,68 @@ def save(self, *args, **kwargs):
 
     class Meta:
         ordering = ('created',)
+
+
+class RRset(models.Model):
+    # TODO Do these two fields really make sense? Meaning is limited when deleting + recreating an RRset
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(null=True)
+    domain = models.ForeignKey(Domain, on_delete=models.CASCADE, related_name='rrsets')
+    subname = models.CharField(max_length=178, blank=True)
+    type = models.CharField(max_length=10)
+    records = models.CharField(max_length=64000, blank=True)
```

Comment:
Please consider [this comment](https://github.com/desec-io/desec-stack/pull/50/files/e24690761137d31315045721ac1716b71fff7600#r123376800).